src/blog/2023/02/flowforge-1-4-0-released.md

```diff
@@ -0,0 +1,102 @@
+---
+title: FlowForge v1.4 with device provisioning in bulk and staged development process
+subtitle: Our second release of 2023 with some great new features to try out.
+description: Deploy Node-RED to many devices quickly, and allow a staged development process with the latest release of FlowForge v1.4.
+date: 2023-02-16 14:00:00.0
+authors: ["ian-skerrett"]
+video: vbg4zTmUYjQ
+image: /blog/2023/0102/images/ff-r14-image.png
```

Comment:
@iskerrett looks like the file path is wring, should just be 02/ according to where the image was added